mscore/scoreaccessibility.cpp

```diff
@@ -249,22 +270,28 @@ void ScoreAccessibility::updateAccessibilityInfo()
       //getInspector->isAncestorOf is used so that inspector and search dialog don't loose focus
       //when this method is called
       //TODO: create a class to manage focus and replace this massive if
-      if ( (qApp->focusWidget() != w) &&
-           !mscore->inspector()->isAncestorOf(qApp->focusWidget()) &&
-           !(mscore->searchDialog() && mscore->searchDialog()->isAncestorOf(qApp->focusWidget())) &&
-           !(mscore->getSelectionWindow() && mscore->getSelectionWindow()->isAncestorOf(qApp->focusWidget())) &&
-           !(mscore->getPlayPanel() && mscore->getPlayPanel()->isAncestorOf(qApp->focusWidget())) &&
-           !(mscore->getSynthControl() && mscore->getSynthControl()->isAncestorOf(qApp->focusWidget())) &&
-           !(mscore->getMixer() && mscore->getMixer()->isAncestorOf(qApp->focusWidget())) &&
-           !(mscore->searchDialog() && mscore->searchDialog()->isAncestorOf(qApp->focusWidget())) &&
-           !(mscore->getDrumrollEditor() && mscore->getDrumrollEditor()->isAncestorOf(qApp->focusWidget())) &&
-           !(mscore->getPianorollEditor() && mscore->getPianorollEditor()->isAncestorOf(qApp->focusWidget()))) {
+      QWidget* focusWidget = qApp->focusWidget();
```

Comment:
FWIW, this change is a no-op functionally, but it is more efficient, and it aids in further debugging